logic/settings.ts

```diff
@@ -105,6 +108,26 @@ export const COUNT_TYPES = [
 	CountType.FileSize,
 ];
 
+const NOTE_COUNT_TYPES = [
+	CountType.None,
+	CountType.Word,
+	CountType.Page,
+	CountType.PageDecimal,
+	CountType.ReadTime,
+	CountType.PercentGoal,
+	CountType.Note,
+	CountType.Character,
+	CountType.Link,
+	CountType.Embed,
+	CountType.Alias,
+	CountType.Created,
+	CountType.Modified,
+	CountType.FileSize,
+	CountType.CustomKey,
+];
+
+export const COUNT_TYPES = [...new Set([...FOLDER_COUNT_TYPES, ...NOTE_COUNT_TYPES])];
```

Comment:
I want to think more about how frontmatter keys might aggregate up to folders, rather than splitting the count types up like this. Maybe the best way would be to show the number of child notes where the frontmatter key is populated.

Reply:
I have already made modifications to several other places. In my opinion, the behavior of displaying the number of child notes in the folder and the frontmatter key in the note is completely inconsistent. If there is no frontmatter key, the best approach is not to display it. If you insist, I completely respect your design. Please tell me how to modify it specifically